schema/field_of_view/tiles/tiles.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "schema/field_of_view/tiles/tiles.json",
+  "description": "Specification of a 2-d image tile.",
+  "type": "object",
+  "required": [
+    "coordinates",
+    "indices",
+    "file",
+    "sha256"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "coordinates": {
+      "$ref": "coordinates.json"
+    },
+    "indices": {
+      "$ref": "indices.json"
+    },
+    "file": {
+      "type": "string",
+      "description": "A relative or absolute location of the image file containing data for this tile.",
+      "example": "img_000000000_Bandpass488_000.tif"
+    },
+    "sha256": {
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9]{64}$",
+      "description": "The Sha256 hash result for the file corresponding to this tile.",
+      "example": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+    },
+    "tile_format": {
+      "type": "string",
+      "description": "The file format of this tile.",
+      "enum": [
+        "TIFF",
+        "NUMPY"
+      ]
+    },
+    "tile_shape": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "type": "integer",
+        "description": "The shape (x, y) of this tile.",
+        "minimum": 0
+      }
+    },
+    "extras": {
```

Comment:
@hewgreen @malloryfreeberg I think I've messed up relative linking in this spec. I tried to pattern off of the `humancellatlas/metadata-schema` repo. What I'm most concerned about is the interaction between the testing code and the schemas. 

When I reference a sub-schema, it uses a relative path. With the current pattern I'm not able to reference a file that is in a higher directory. For example, replacing the code block associated with this comment with a link to `extras.json` does not work:

```python
self = <jsonschema.validators.RefResolver object at 0x1052af978>, url = 'file://<root>/sptx-format/schema/field_of_view/tiles/schema/extras.json'

    def resolve_from_url(self, url):
        url, fragment = urldefrag(url)
        try:
            document = self.store[url]
        except KeyError:
            try:
                document = self.resolve_remote(url)
            except Exception as exc:
>               raise RefResolutionError(exc)
E               jsonschema.exceptions.RefResolutionError: <urlopen error [Errno 2] No such file or directory: '<root>/sptx-format/schema/field_of_view/tiles/schema/extras.json'>
```

You can see that each time an import happens the path gets appended to the prior root, which doesn't resolve to a correct path after the "first resolving". Is this something you've run into before?